spack/packages/dla-future/package.py

```diff
@@ -25,7 +25,9 @@ class DlaFuture(CMakePackage, CudaPackage):
     depends_on("mpi")
     depends_on("blaspp")
     depends_on("lapackpp")
-    depends_on("hpx cxxstd=14 networking=none")
+    depends_on("umpire~examples")
+    depends_on("umpire+cuda~shared", when="+cuda")
+    depends_on("hpx cxxstd=14 networking=none +async_mpi")
```

Comment:
NB. I added `+async_mpi` here. If someone wants it I can make a separate PR for that.